taxi/views.py

```diff
@@ -64,7 +88,8 @@
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarForm
+    template_name = "taxi/car_form.html"
     success_url = reverse_lazy("taxi:car-list")
```

Comment:
The checklist specifies that URLs should end with a '/'. Please add a '/' at the end of the URL in `success_url`.